conf/application.conf

```diff
@@ -97,6 +97,7 @@ api {
 
   # The context which the API will have via the API Platform http://API_GATEWAY/{api.gateway.context}/
   gateway.context = "individuals/self-assessment-test-support"
+  businessDetails.gateway.context = "individuals/business/details"
```

Comment:
Would it be worth moving this out of this section (as it is not for this API). Maybe could add another section?